### ISSUE TITLE
Get strip-components PR merged into camptocamp-archive

### ISSUE THREAD

**Issue by dfarrell07** [dfarrell07/puppet-opendaylight#57]
I added the ability to pass the `--strip-components` flag to `camptocamp-archive`'s tar extract step. Need to get [my fork](https://github.com/camptocamp/puppet-archive) merged into the main project.

This is kind-of a blocker for merging the `archive_mod` branch into `master`, as Beaker can't `puppet module install` on host VMs from a git repo (only released mods on the Forge). Therefore, I can't point my Beaker tests at my pre-PR fork and use that until the PR is merged.

The code-level change in the PR seems to be solid. I've tested it on the `archive_mod` branch of [vagrant-opendaylight](https://github.com/dfarrell07/vagrant-opendaylight/).

Need to update the `camptocamp-archive` mod's docs to reflect the API change before I can submit the PR.

**Comment by dfarrell07** [comment]
> Beaker can't puppet module install on host VMs from a git repo (only released mods on the Forge)

Note that I'm actively working with the Puppet community to fix either the PMT or Beaker to support this and similar use-cases. [Here's a relevant Jira ticket](https://tickets.puppetlabs.com/browse/FORGE-50).

**Comment by dfarrell07** [comment]
> Need to update the camptocamp-archive mod's docs to reflect the API change before I can submit the PR.

Updated the docs in [this commit](https://github.com/camptocamp/puppet-archive/commit/53f997c14e2cbc5401e9c7ffb160e0748420e5cd).

> Need to get my fork merged into the main project.

[Submitted a PR](https://github.com/camptocamp/puppet-archive/pull/40), with a long and updated version of my [notes](https://gist.github.com/dfarrell07/e5898d01fb94b5ded6d1) on the change.

Now just waiting on the review and (hopefully) merge.

**Comment by dfarrell07** [comment]
Separating the rake tasks for our Beaker tests (#54: `rake centos_tarball` for a tarball install, `rake centos` is still an RPM install) made this a bit less pressing, as the RPM install Beaker tests are no longer broken by in-dev tarball install Beaker tests (#45).

**Comment by dfarrell07** [comment]
This is blocking #45, which is very painful. Can't create Beaker tests for the tarball install method until it's merged.

**Comment by dfarrell07** [comment]
I [pinged my unmerged PR's thread](https://github.com/camptocamp/puppet-archive/pull/40#issuecomment-76792362), hoping to get some attention. No luck so far. Labeled this as `blocked` because it can't make progress without action from upstream `camptocamp-archive` folks.

**Comment by dfarrell07** [comment]
[It was merged](https://github.com/camptocamp/puppet-archive/pull/40#event-247342999)! :)
